api/src/main/java/com/okta/sdk/resource/user/UserBuilder.java

```diff
@@ -51,6 +51,8 @@ static UserBuilder instance() {
 
     UserBuilder setProvider(Boolean provider);
 
+    UserBuilder setType(UserType userType);
```

Comment:
I'm not 100% sure on how this would be used, but you may want to also add a `UserBuilder setType(String userTypeId)` I'm [only] guessing, but the caller of this may not have an instance of `UserType`

(just an idea, 🧂)

Reply:
Added an overloaded `setType(String userTypeId)` method in `UserBuilder`.